src/styles/header.scss

```diff
@@ -0,0 +1,242 @@
+.header {
+  background-image: url(../image/girl.png);
+  background-size: cover;
+  background-position: center;
+  height: 100vh;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  &_logo {
+    display: flex;
+  }
+
+  &__img {
+    width: 125px;
+    height: 16px;
+
+    @include onTablet {
+      height: 24px;
+      width: 127px;
+
+    }
+  }
+
+  &__icons {
+    display: flex;
+    align-items: center;
+  }
+
+  &__bottom {
+    padding-bottom: 40px;
+    @include contentPaddingInline;
+
+    @include onTablet {
+      padding-bottom: 65px;
+    }
+  }
+
+  &__title {
+    font-size: 36px;
+    font-weight: 700;
+
+    @include onTablet {
+      font-size: 48px;
+    }
+
+    @include onDesktop {
+      font-size: 56px;
+    }
+
+  }
+}
+
+.icon {
+  height: $icone-size;
+  width: $icone-size;
+  display: block;
+  transition: transform 0.3s;
+
+  @include onTablet {
+    height: 32px;
+    width: 32px;
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  @include onDesktop {
+    @include hover(transform, scale(1.2));
+  }
+
+  &__phone-number {
+    color: $main-color;
+    font-size: 12px;
+    letter-spacing: 2px;
+
+    opacity: 0;
+    transition: opopacity 0.3s;
```

Comment:
```suggestion
    transition: opacity 0.3s;
```

Reply:
Be careful, because grammatical errors will cause the code to not work as you expect.